### DISCUSSION TITLE
Skywalking Zipkin Lens UI not visible in Apache skywalking 9.4.0

### DISCUSSION THREAD

**Discussion by skundu1234** [apache/skywalking#10834] [General]
Once we installed Skywalking UI and OAP ,plulling from docker hub.We are not able to see that Zipkin lens ui.Can you please tell that its alreday released or we need to wait till next version.
<img width="1785" alt="MicrosoftTeams-image (2)" src="https://github.com/apache/skywalking/assets/87635275/de12e6c4-f3da-40a5-8b75-0fe79d656c87">

**Comment by wu-sheng** [comment]
Check release version, it is documented clearly.

**Reply by skundu1234** [reply]
Thanks @wu-sheng  .Its clearly mentioned but it's not visible zipkin lens UI so asking.

**Reply by wu-sheng** [reply]
I can see that on demo and showcase. So, can't tell what happens at you private env.

**Reply by wankai123** [reply]
Under menu: `Service Mesh --> Services`.
